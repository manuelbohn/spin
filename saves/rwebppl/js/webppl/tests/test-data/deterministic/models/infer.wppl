var model = function() {
  return flip(1);
};
[
  // Test the use of ES6 object literal shorthand.
  sample(Infer({model})),
  sample(Infer({model: model})),
  sample(Infer({model() { return model(); }})),
  // Test backward compatibility with previous interface.
  sample(Infer({}, model))
];
