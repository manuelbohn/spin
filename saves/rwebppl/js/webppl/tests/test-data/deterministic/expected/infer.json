{
  "result": [true, true, true, true]
}
